build_scripts/build_agent_linux.sh

```diff
@@ -0,0 +1,99 @@
+#! /bin/bash
+#
+SCRIPT_NAME="$(basename "$0")"
+
+BRANCH="develop"
+LOCAL=false
+DIST_DIR="${PWD}/agent-dist"
+
+die() {
+    echo "$1" >&2
+    echo ""
+    echo_help
+    exit 1
+}
+
+echo_help() {
+  echo "Builds an Infection Monkey agent for Linux"
+  echo ""
+  echo "Usage:"
+  echo "    ${SCRIPT_NAME}"
+  echo "    ${SCRIPT_NAME} --branch <BRANCH>|--local"
+  echo "    ${SCRIPT_NAME} -h|--help"
+  echo ""
+  echo "Options:"
+  echo "    --branch                   Branch to build from. Default: ${BRANCH}"
+}
+
+while (( "$#" )); do
+  case "$1" in
+    -h|--help)
+      echo_help
+      exit 0
+      ;;
+    --branch)
+      BRANCH=$2
+      shift 2
+      ;;
+    --local)
+      LOCAL=true
+      shift
+      ;;
+    *)
+      die "Error: Unsupported parameter \"$1\"."
+      ;;
+  esac
+done
+
+mkdir -p "$DIST_DIR"
+
+setup_environment_commands="
+set +x &&
+export PYENV_ROOT=\"\${HOME}/.pyenv\" &&
+command -v pyenv >/dev/null || export PATH=\"\$PYENV_ROOT/bin:\$PATH\" &&
+eval \"\$(pyenv init -)\" &&
+python --version &&
+pip install pipenv &&
+"
+
+clone_commands="
+echo \"Cloning branch ${BRANCH}...\" &&
+git clone https://github.com/guardicore/monkey.git -b \"${BRANCH}\" --single-branch --depth 1 &&
+cd monkey/monkey/infection_monkey &&
+"
+
+local_commands="
+cd /src/monkey/infection_monkey &&
```

Comment:
Should this path be configurable? Maybe make `--local` take a path as an argument?

Reply:
My thought was to keep this option as simple as possible. When I specify it, I don't want to also have to specify a location. See the updated documentation for the script and let me know your thoughts.

Reply:
Oh I see, I thought this script was in another repo. This command is run inside the container, where `/src/monkey` _is_ this repo